vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/experiment/ExperimentCreationDialog.java

```diff
@@ -116,14 +120,41 @@ private HorizontalLayout createExperimentalVariableLayout() {
     return rowLayout;
   }
 
+  @Override
+  public void open() {
+    throw new UnsupportedOperationException("open without project id not supported");
+  }
```

Comment:
Question: Can't the projectId be retrieved from the URL instead of overriding the basic functionality? This would lead us to always have to define the projectID

Reply:
Yes :) sadly not yet. The goal would be to do so, yes.